binding/web/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@picovoice/orca-web",
+  "description": "Orca Text-to-Speech engine for web browsers (via WebAssembly)",
+  "author": "Picovoice Inc",
+  "license": "Apache-2.0",
+  "version": "0.1.2",
```

Comment:
?

Reply:
Should it be a different version? I just saw that for Python it seemed like it was v 0.1.2

Reply:
All package versions should start from 0.1.0 since Orca is in beta. The first two values denote the version of the engine, while the last one is reserved for SDK-level patching. For Python, we have already released two patches, hence it is labeled as 0.1.2. Does it make sense?

Reply:
got it, that makes sense